features/2025-07-radius-resource-types-contribution.md

```diff
@@ -0,0 +1,343 @@
+      Audience: Developers/Contributors exploring new technologies or learning Radius
+      Requirements:
+       - Resource type schema Validation: YAML schema passes validation
+       - Single Recipe: At least one working recipe for any cloud provider or platform
+       - Basic Documentation: README with usage examples
+       - Manual Testing: Evidence of local testing by contributor
+
+    _Stage 2 : Verified(Beta)_
+
+      Purpose: Ensure contributions meet production-ready standards with comprehensive testing and documentation
+      Audience: Contributors seeking to have their resource types included in official Radius releases
+      Requirements:
+       - Multi-Platform Support: Recipes for all three platforms ( AWS, Azure, Kubernetes)
+       - IaC Support: Recipes for both Bicep and Terraform
+       - Automated Testing: Functional tests that validate resource type and Recipes
+       - Documentation: Detailed README with Recipe coverage, troubleshooting guides, and best practices
+       - Ownership: Designated owner for the resource type and Recipe
+       - Maintainer Review: Formal review and approval by Radius maintainers
+
+    _Stage 3 : Production Ready(Stable)_
+
+      Purpose: Establish Resource types and Recipes as officially supported and maintained by the Radius project
+      Audience: Enterprise users doing production deployments and seeking stable, well-tested Resource types and Recipes
+      Requirements:
+       - Functional tests have 100% coverage and results for Resource type schema and Recipe
+       - Integration Testing: Full integration with Radius CI/CD pipeline and release process
+       - Documentation: Complete user guides, troubleshooting, and best practices
+       - SLA Commitment: Defined support level and response time commitments
+       - Maintainer Review: Formal review and approval by Radius maintainers
+    
+    Contributor has to follow the contribution guidelines and the checklist for each stage to ensure that their contribution meets the requirements. Radius maintainers review the contribution meet the bar of quality for each of these stages before merging the contribution into the `resource-types-contrib` repository.
+
+- Should we ship all resource types in this repository as part of Radius ?
+
+  A Resource type cannot be useful without a Recipe. Every time a new resource type is added to the `resource-types-contrib` repository, it must be accompanied by at least one default Recipe that works with the resource type. Shipping just the resource type will impact the developer experience as it will cause deployment errors when there is no default Recipe available for the resource type. Based on the stages defined, we ship only resource types that have reached the `stable` stage as part of Radius. Community is encouraged to use the alpha and beta resource types and Recipes from the `resource-types-contrib` repository.
```

Comment:
Who decides when a recipe is stable? How is stable vs. other statuses indicated?